homeassistant/components/wemo/__init__.py

```diff
@@ -69,6 +70,7 @@ def coerce_host_port(value: str) -> HostPortTuple:
     {
         DOMAIN: vol.Schema(
             {
+                vol.Optional(CONF_CALLBACK_ADDRESS): cv.string,
```

Comment:
Shouldn't this be provided by the Network integration instead?

Additionally:

For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1:

Reply:
I don't see how this would go in the Network integration... it configures a pywemo-specific environment variable. Anyway, will rework after config flow is done. Beyond my current capabilities.